packages/tree-select/test/index.js

```diff
@@ -4,7 +4,7 @@ import { filter } from 'lodash';
 /**
  * Internal dependencies
  */
-import treeSelect from '../';
+import treeSelect from '../src';
```

Comment:
Maybe we can drop this change when we get bootstrap sorted out: https://github.com/Automattic/wp-calypso/pull/27627#discussion_r234126319

Reply:
i'm ok with keeping it. fewer ordering issues seems sane.